tools/depmod.c

```diff
@@ -47,11 +47,12 @@ static const char *const default_cfg_paths[] = {
 	NULL
 };
 
-static const char cmdopts_s[] = "aAb:o:C:E:F:evnP:wVh";
+static const char cmdopts_s[] = "aAb:m:o:C:E:F:evnP:wVh";
 static const struct option cmdopts[] = {
 	{ "all", no_argument, 0, 'a' },
 	{ "quick", no_argument, 0, 'A' },
 	{ "basedir", required_argument, 0, 'b' },
+	{ "moduledir", required_argument, 0, 'm' },
```

Comment:
On the kernel side we will probably require using the long option, `--moduledir.` Otherwise we won't have a depmod failure due to having a previous use for `-m`.

Reply:
Or check kmod version, or add yet another flag to pkgconfig.